lib/ash/filter/filter.ex

```diff
@@ -872,7 +875,10 @@ defmodule Ash.Filter do
         value
 
       {:error, error} ->
-        raise Ash.Error.to_ash_error(error)
+        raise Ash.Error.to_ash_error(error, nil,
+                error_context:
+                  "parsing #{inspect(base)} #{inspect(addition)} #{inspect(op)} #{inspect(aggregates)} #{inspect(calculations)} in #{inspect(context)}"
```

Comment:
Another placeholder - will need to put in a useful context here.

Reply:
Same comment as above